Podfile

```diff
@@ -35,10 +35,10 @@ abstract_target 'CI' do
   pod 'SwiftLint', swiftlint_version
 end
 
-post_install do |pi|
-  pi.pods_project.targets.each do |t|
-    t.build_configurations.each do |config|
-      if t.name != 'Pocket Casts Watch App'
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      if target.name != 'Pocket Casts Watch App'
```

Comment:
The true reason for this refactor is to update the `Podfile.lock` so that CI will generate a new cached `Pods/` archive which will include the `xcconfig` for the new Prototype build configuration.

Otherwise, CI will download a `Pods/` folder version that does not include the files for that new build configuration and will fail to build with errors such as:

> error: Unable to load contents of file list: '/Target Support Files/Pods-podcasts/Pods-podcasts-frameworks-Prototype-output-files.xcfilelist' (in target 'podcasts' from project 'podcasts')

See
https://buildkite.com/automattic/pocket-casts-ios/builds/7090#018fe6b5-508b-443b-abb9-064a04ae0054